docs/UserGuide.md

```diff
@@ -102,37 +104,42 @@ PropertyWhiz has been extensively tested on Microsoft Windows 10 and Ubuntu 20.0
 
     * **`exit`** : Exits the app.
 
+
 6. Refer to the [Commands](#commands) below for details of each command, 
    or the [Command Summary](#command-summary).
 
+
 --------------------------------------------------------------------------------------------------------------------
 
 
 ## Command summary
 
+
 Refer to the section [Command Syntax Guide](#command-syntax-guide) below for how to interpret our commands, 
+
 or [Commands](#commands) for a more detailed version of each action.
 
 Action | Format, Examples
 --------|------------------
 **Add** | **Property** <br>`add property n/PROPERTY_NAME a/PROPERTY_ADDRESS s/SELLER_NAME p/SELLER_PHONE e/SELLER_EMAIL $/PRICE [t/TAG]…​` <br><br> **Buyer** <br>`add buyer n/BUYER_NAME p/BUYER_PHONE e/BUYER_EMAIL $/BUDGET) [t/TAG]…`
 **Clear** | `clear`
-**Delete** | `delete (property | buyer) INDEX`
-**Edit** | **Property** <br>`edit property INDEX [n/PROPERTY_NAME] [a/PROPERTY_ADDRESS] [s/SELLER_NAME] [p/SELLER_PHONE] [e/SELLER_EMAIL] [$/PRICE] [([t/TAG]…​ | [ta/TAG_TO_ADD]… [td/TAG_TO_DELETE]…)]]​`<br><br> **Buyer** <br> `edit buyer INDEX [n/BUYER_NAME] [p/BUYER_PHONE] [e/BUYER_EMAIL] [$/BUDGET]) [([t/TAG]… | [ta/TAG_TO_ADD]… [td/TAG_TO_DELETE]…)]`
-**Find** | `find (property | buyer) [KEYWORDS] [t/TAG_TO_MATCH]…`
+**Delete** | `delete ( property | buyer ) INDEX`
```

Comment:
I believe we are removing the spaces, see Elizabeth's PR. Likewise for other cases.
i.e. `delete (property | buyer) INDEX` rather than `delete ( property | buyer ) INDEX`

Reply:
Alright noted and removed